WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverViewModel.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import UIKit
+import WordPressUI
+
+final class CompliancePopoverViewModel: ObservableObject {
+    @Published
+    var isAnalyticsEnabled: Bool = true
```

Comment:
🟠 I understand that this popup will only show up once, but in my testing, I forced `shouldShowCompliancePopup` to return `true` and noticed the analytics switch is always `true`. See video.

<video src="https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/83d2b0c2-17b2-4930-bcfa-3f660f0745b9"></video>

Reply:
It will sound funny but I kind of left it there as a feature 😂 
I can definitely update it as per your suggestion but hear me out:

If a user has already disabled this, I don't think it even makes sense to show such a pop-up. But, it would give us another chance to ask them (also by legal terms we have to ask it in either case I guess?) to enable it. If we keep the default to enabled, and they simply don't care, they can tap on save and let the analytics function. If they do care and they don't want us to do it, they would take the active action and switch the toggle off again.

Some people find this manipulative which is understandable. I mostly feel like some people (I do this often too) just want to get rid of the pop up and don't really care about the context. In that case, I think it would be beneficial to be able to collect the data. As it is only analytics data that just helps us tailor the app for users' needs.

Wdyt? If it sounds odd, I will apply your suggestion :D

Reply:
@alpavanoglu I see, thanks for clarifying. Now that you explained this, I don't think it's a bad idea. I was also concerned that users might disable tracking even though we only use Analytics to improve the app. That being said, I'm a bit torn about what's the best way to go, perhaps we should just align with Android or Woo, what do you think?

Reply:
I know woo default to the chosen option. I am not sure what Android has done.